capsule-util/src/main/resources/capsule/trampoline-execheader.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec `java -Dcapsule.trampoline -jar $0` "$@"
+exec bash -c "exec $(java -Dcapsule.trampoline -jar $0) $@"
```

Comment:
If you have quotes around the command, it runs if you type it into a shell

```
$ 'echo' 'test'
test
```

However, if you use command substitution then the quotes are cause the following error:
```
$ CMD='"echo" "date"'
$ `$CMD`
bash: "echo": command not found
```

This can be addressed by calling `bash -c`, which executes the command as if you'd typed it into the shell. Having the inner command be an `exec` as well makes sure we don't leave an extra bash process lying around.